front/init.js

```diff
@@ -8,6 +8,8 @@ const { execSync } = require('child_process')
 
 let URLPATH = process.argv[2] || '.'
 if (URLPATH.endsWith('/')) URLPATH = URLPATH.slice(0, -1)
+let publicBinOnly = process.argv[3] === true
```

Comment:
I may be wrong, but I don't think that a process.argv would evaluate to strict `true`, it would be string. I think it's better to do as `const publicBinOnly = process.argv.includes('--publicBinOnly')`

Reply:
Good catch. I've changed it as you suggested.

Also, when I run the script I get:

--- !!! unable to reset the mtime for the extracted proteinpaint bundle:  ReferenceError: codepath is not defined

Was the declaration of `codepath` deleted recentry?